docs/reference/ml/apis/start-dfanalytics.asciidoc

```diff
@@ -0,0 +1,57 @@
+[role="xpack"]
+[testenv="platinum"]
+[[start-dfanalytics]]
+=== Start {dfanalytics-jobs} API
+
+[subs="attributes"]
+++++
+<titleabbrev>Start {dfanalytics-jobs}</titleabbrev>
+++++
+
+beta[]
+
+Starts one or more {dfanalytics-jobs}.
+
+[[ml-start-dfanalytics-request]]
+==== {api-request-title}
+
+`POST _ml/data_frame/analytics/<data_frame_analytics_id>/_start`
+
+[[ml-start-dfanalytics-prereq]]
+==== {api-prereq-title}
+
+* You must have `machine_learning_admin` built-in role to use this API. You must 
+also have `read` and `view_index_metadata` privileges on the source index and 
+`read`, `create_index`, and `index` privileges on the destination index. For 
+more information, see {stack-ov}/security-privileges.html[Security privileges] 
+and {stack-ov}/built-in-roles.html[Built-in roles].
+
+[[ml-start-dfanalytics-path-params]]
+==== {api-path-parms-title}
+
+`<data_frame_analytics_id>` (Required)::
+  (string) Identifier for the {dfanalytics-job}. This identifier can contain
+  lowercase alphanumeric characters (a-z and 0-9), hyphens, and underscores. It
+  must start and end with alphanumeric characters.
+
```

Comment:
Similarly to start-datafeed API, this one may also accept a `timeout` parameter.